modules/ROOT/pages/software-extensibility.adoc

```diff
@@ -57,7 +57,7 @@ Custom REST API extensions developed with the Maven archetype are added to a Bon
 
 In Community, Teamwork, Efficiency, Performance and Enterprise editions
 
-A xref:pages.adoc[custom page] is a page that you can add into Bonita Portal to form part of an application, customize information provided in default portal pages, or to add new features to the portal.
+A xref:pages.adoc[custom page] is a page that you can add into Bonita Administrator Application to form part of an application or to customize information provided in default Application pages.
 To implement a page, you need to provide HTML, CSS and Javascript resources respecting some packaging constraints.
 You can create a page using the xref:ui-designer-overview.adoc[UI designer], which automatically creates a well-formed page.
 Note that although the page framework provided in the product is stable, we cannot guarantee that all custom pages will work with future versions, because this depends on the details of how the page is implemented.
```

Comment:
on the details of how the page is implemented. => on how the page was implemented details.

Reply:
I modified it to on how the page was implemented. New phrase: 
"Note that although the page framework provided in the product is stable, we cannot guarantee that all custom pages will work with future versions, because this depends on how the page was implemented."

Reply:
Is this ok @vbotnari12 ?

Reply:
Well, I updated it. Hopefully it's fine